GetJobber/atlantis
# build: Update how we handle jobber fonts

## Motivations



## Changes



### Added

- 

### Changed

- Changed shrinkwrap back to package-lock
- Separate concern with fonts and design
  - `design` will reference jobber pro in the CSS and it it exists, it'll use it, otherwise, it won't (native css functionality)
  - `fonts` would give you a css file that you can import to your project if you want to use it. That enables the CSS to find the font
    - This follows the same paradigm as importing `foundation.css` from `@jobber/design` to your [repo manually](https://atlantis.getjobber.com/packages/design/#css).
- This change hopes to keep our package simple with nothing to a minimal script running after installation since it should just work.

### Deprecated

- 

### Removed

- 

### Fixed

- 

### Security

- 

## Testing

- Install the pre-release below on your repo. Including the [extra step for design](https://atlantis.getjobber.com/packages/design/#css)
- Run the components
  - It should still use Poppins and Open Sans as the font
- Install @jobber/font and import the css that comes with it to your project. See `gatsby-browser.js`
  - It should now use Jobber Pro and Inter

## Prerelease

```
 - @jobber/components@3.0.3-pre.2+a045d54b
 - @jobber/design@0.28.3-pre.2+a045d54b
 - @jobber/docz-tools@2.0.3-pre.2+a045d54b
```

---

[In Atlantis we use Github's built in pull request reviews](https://help.github.com/en/articles/about-pull-request-reviews).

![Random photo of Atlantis](https://loremflickr.com/672/400/atlantis)

JOB-62076
